### ISSUE TITLE
Getting error after installation 

### ISSUE THREAD

**Issue by MaryGoAround** [dnanexus/dx-toolkit#1313]
Hi
 
I need help with `dx-toolkit` please.
 
I guess I have installed that on my local machine.
 
```
Last login: Fri Jan 19 16:27:11 on ttys000
user@LV19Y7325V ~ % pip3 install --upgrade dxpy
Requirement already satisfied: dxpy in /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages (0.369.0)
Requirement already satisfied: websocket-client==0.54.0 in /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages (from dxpy) (0.54.0)
Requirement already satisfied: python-dateutil>=2.5 in /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages (from dxpy) (2.8.2)
Requirement already satisfied: psutil>=5.9.3 in /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages (from dxpy) (5.9.7)
Requirement already satisfied: certifi in /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages (from dxpy) (2023.11.17)
Requirement already satisfied: urllib3<2.2,>=1.25 in /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages (from dxpy) (2.1.0)
Requirement already satisfied: argcomplete>=2.0.0 in /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages (from dxpy) (3.2.1)
Requirement already satisfied: six in /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages (from websocket-client==0.54.0->dxpy) (1.1
 
```
But I get error when look for any `dx` functioning like
 
```
user@LV19Y7325V ~ % dx help ls                 
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/bin/dx", line 5, in <module>
    from dxpy.scripts.dx import main
  File "/Users/user/Downloads/dx-toolkit-master/src/python/dxpy/scripts/dx.py", line 35, in <module>
    from dxpy.scripts import dx_build_app
  File "/Users/user/Downloads/dx-toolkit-master/src/python/dxpy/scripts/dx_build_app.py", line 40, in <module>
    from dxpy.nextflow.nextflow_builder import build_pipeline_from_repository, prepare_nextflow
  File "/Users/user/Downloads/dx-toolkit-master/src/python/dxpy/nextflow/nextflow_builder.py", line 15, in <module>
    from distutils.dir_util import copy_tree
ModuleNotFoundError: No module named 'distutils'
```
 
 
Please, help me to see what is going on here.
 
Thanks a million, in advance

**Comment by sclan** [comment]
"Requirement already satisfied: dxpy in /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages (0.369.0)"

0.369.0 is not released into pypi yet. What happens if you install the package that is not found?
`pip3 install --upgrade pip; pip3 install distutils`

**Comment by wormsik** [comment]
Dear @MaryGoAround , this issue is probably related to Python 3.12 which dropped `distutils` package from the standard library. We addressed it in dx-toolkit version `0.370.2` which is available in PyPI. If the problem is not resolved, please reach out to us again. Best, Jan.
